static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -236,6 +236,10 @@ $fc-item-gutter: $gs-gutter / 4;
         .fc-item__timestamp {
             flex: 1;
         }
+        
+        .fc-item__standfirst {
+            padding-bottom: $gs-gutter * .25;
+        }
```

Comment:
This adds a padding-bottom to the standfirst of all facia cards when there's a timestamp. Fixing: https://trello.com/c/muGkgPKH/249-opinion-cards-on-tag-pages-padding-issue